frappe/public/js/frappe/views/kanban/kanban_card.html

```diff
@@ -1,9 +1,17 @@
 <div class="kanban-card-wrapper {{ disable_click }}" data-name="{{name}}">
-	<div class="kanban-card content">
-		<div class="kanban-card-title">
-			{{ title }}
+	<a class="kanban-card-redirect" href="#" target="_blank">
+		<div class="kanban-card content">
+			<div class="kanban-card-title">
+				<!-- Hack to retrieve task document number,
+					since TASK00001 looks bad on the card -->
+				<div class="text-muted">
+					<span>#{{ flt(name.split("TASK")[1]) }}</span>
```

Comment:
I don't think we should do this. This code is specific to Task doctype. We should keep it generic.

Reply:
Rectified in the latest commit.